search/includes/classes/class-search.php

```diff
@@ -870,13 +880,15 @@ public function ep_handle_failed_request( $response, $query, $statsd_prefix ) {
 			$error_message = $response_error['reason'] ?? 'Unknown Elasticsearch query error';
 			$this->logger->log(
 				'error',
-				'vip_search_query_error',
+				'search_query_error',
 				$error_message,
 				[
 					'error_type' => $response_error['type'] ?? 'Unknown error type',
 					'root_cause' => $response_error['root_cause'] ?? null,
 					'query' => $query_for_logging,
 					'backtrace' => wp_debug_backtrace_summary(),
+					'is_cli' => $is_cli,
+					'request' => $encoded_request,
```

Comment:
How do feel about adding the whole $response_body  here? I feel like we are getting these `Unknown Elasticsearch query error` quite a lot.

Or maybe add the body if there is no ` $response_error['reason']`? To avoid pollution?

Reply:
I think we're fine by adding all the response all times. Actually, we're doing this PR especially for those unknown errors.